src/vulnpy/common/utils.py

```diff
@@ -0,0 +1,16 @@
+import os
```

Comment:
so I think my only ask is that we not name this file utils.py. Mostly because I've noticed us ending up keeping a utils.py file growing and growing, and these files don't have much architectural meaning. Given that `get_template` is really the only meaningful thing right now, you could put the function in the __init__ file (which I know isn't your personal pref) or rename utils.py to .... idk, template.py?

Reply:
it's funny you say that because I had a bit of a strange feeling doing it. I'll rename the file - I have an import alias set up in `__init__.py` so the function will still be importable as
```
from vulnpy.common import get_template
```

- [ ] rename utils.py